### PR TITLE
Add gs system schemas

### PR DESCRIPTION
**Pull request by tshemsedinov**

### CODE REVIEW

File: schemas/system/Identifier.schema [hunk 1]
```diff
@@ -0,0 +1,7 @@
+System({
+  Family: Enum('Registry', 'Dictionary', 'File', 'Cache'),
+  Status: Enum('Preallocated', 'Intialization', 'Actual', 'Historical'),
+  Lock:  Enum('Free', 'Locked'),
+  Create: { domain: 'Timestamp' },
```
Comment (lundibundi) [inline schemas/system/Identifier.schema]: Maybe `Created` and `Changed`? As I assume they denote creation/last change time?

File: schemas/system/Identifier.schema [hunk 2]
```diff
@@ -0,0 +1,7 @@
+System({
+  Family: Enum('Registry', 'Dictionary', 'File', 'Cache'),
```
Comment (lundibundi) [inline schemas/system/Identifier.schema]: Maybe `Type` or similar? `Family` looks strange to me here, any specific reasons for such naming?

**Comment by lundibundi** [comment]
Could you please elaborate on the meaning of `System(..)` and `Parent(..)` (I've seen them in decorators but what's the meaning of them)?

File: schemas/system/Identifier.schema (hunk 1 above)
Comment (tshemsedinov) [reply schemas/system/Identifier.schema]: No, in databases we use singular infinitive in entity and attribute naming. In *nix systems we have access, change, modify time, not accessed, changed, modified time.

**Comment by tshemsedinov** [comment]
Landed in 0858c3f

**Pull request closed without merging**